catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/ApiSwaggerTest.scala

```diff
@@ -11,53 +11,52 @@ class ApiSwaggerTest extends FunSpec with Matchers with fixtures.Server {
 
   it("returns a valid JSON response for all api versions") {
     ApiVersions.values.toList.foreach { version: ApiVersions.Value =>
-      val tree = readTree(s"/test/${version.toString}/swagger.json")
+      val tree = readTree(s"/catalogue/${version.toString}/swagger.json")
 
       tree.at("/host").toString should be("\"test.host\"")
       tree.at("/schemes").toString should be("[\"http\"]")
       tree.at("/info/version").toString should be(s""""${version.toString}"""")
-      tree.at("/basePath").toString should be(
-        s""""/test/${version.toString}"""")
+      tree.at("/basePath").toString should be("")
     }
   }
 
   it("includes the DisplayError model all api versions") {
     ApiVersions.values.toList.foreach { version: ApiVersions.Value =>
-      val tree = readTree(s"/test/${version.toString}/swagger.json")
+      val tree = readTree(s"/catalogue/${version.toString}/swagger.json")
       tree.at("/definitions/Error/type").toString should be("\"object\"")
     }
   }
 
   it("shows only the endpoints for the specified version") {
     ApiVersions.values.toList.foreach { version: ApiVersions.Value =>
-      val tree = readTree(s"/test/${version.toString}/swagger.json")
+      val tree = readTree(s"/catalogue/${version.toString}/swagger.json")
       tree.at("/paths").isObject shouldBe true
       tree
         .at("/paths")
         .fieldNames
         .asScala
-        .toList should contain only ("/works", "/works/{id}")
+        .toList should contain only (s"/catalogue/${version.toString}/works", s"/catalogue/${version.toString}/works/{id}")
```

Comment:
This is due to the fact that swagger now outputs paths relative to the host. This also forced to test using the real "catalogue" prefix because `ApiV1Swagger` and `ApiV2Swagger` are singleton objects that are instantiated when the JVM starts. Tests run in a single jvm so they are created once. The ApiTest specify the "/catalogue/{version}/works" endpoints, which are visible from then onwards in the swagger objects, so they were leaking into this test. So I had to change this test suite to use the real api prefix 😐